registry/zookeeper/registry.go

```diff
@@ -492,3 +494,10 @@ func (r *zkRegistry) IsAvailable() bool {
 		return true
 	}
 }
+func sleepWait(n int) {
+	wait := time.Duration(200*n) * time.Millisecond
+	if wait > 3*time.Second {
```

Comment:
pls define a const on this file header.

const (
    MaxWaitInterval = time.Duration(3e9)
)